lib/rules/eol-last.js

```diff
@@ -5,6 +5,10 @@ import Rule from './_base.js';
 
 export default class EolLast extends Rule {
   parseConfig(config) {
+    if (!this.isStrictMode) {
```

Comment:
Let's also add a note to the rule doc that this rule doesn't apply to embedded templates?

Reply:
I've adapted the rule doc as well! I figured the rule doc doesn't need an explanation, as it already said `files` specifically, which I made even more explicit by saying `template files` instead.